src/Controller/Api/CacheController.php

```diff
@@ -35,10 +35,8 @@
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
-/**
- * @Route("/v1/api/cache", methods={"DELETE"})
- * @IsGranted("ROLE_ADMIN")
- */
+#[Route('/{api_entry_point}/cache', requirements: ['api_entry_point' => 'api/v1|v1/api'], defaults: ['_format' => 'json'])]
```

Comment:
```suggestion
#[Route('/{api_entry_point}/cache', 
    requirements: ['api_entry_point' => 'api/v1|v1/api'], 
    defaults: ['_format' => 'json']),
]
```

Or possibly name even the first argument, for clarity/consistency.

Reply:
I've had that too but currently my IDE does not like multiliners. Have to get rid of it first...